Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -280,6 +304,24 @@ DECLARE_SOA_DYNAMIC_COLUMN(ITSNClsInnerBarrel, itsNClsInnerBarrel, //! Number of
                              }
                              return itsNclsInnerBarrel;
                            });
+DECLARE_SOA_DYNAMIC_COLUMN(ITSNCls001, itsNCls, //! Number of ITS clusters
```

Comment:
I am sorry that we seem not to converge. ;-)

I think now where we have the `v001` namespace, we should move this there and call it `ITSNCls` and not `ITSNCls001`. The same for `ITSClusterMapDyn`. What do you think?